### PR TITLE
Accessibility: Color Contrast Fixes

### PR DESCRIPTION
**Pull request by vancura**
Several a11y color contrast fixes to the IDE and Exception Caught Dialog.

### CODE REVIEW

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ExceptionCaughtDialog.cs [hunk 1]
```diff
@@ -49,6 +49,7 @@ class ExceptionCaughtDialog : Gtk.Window
 		VBox VBox;
 		static readonly Xwt.Drawing.Image WarningIconPixbuf = Xwt.Drawing.Image.FromResource ("toolbar-icon.png");
 		static readonly Xwt.Drawing.Image WarningIconPixbufInner = Xwt.Drawing.Image.FromResource ("exception-outline-16.png");
+		static readonly Gdk.Color ExceptionHeaderBackgroundColor = new Gdk.Color (160, 103, 5);
```
[the remark this answers is not in the document]
Comment (vancura) [reply main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ExceptionCaughtDialog.cs]: Sure, let me fix that.

[the remark this answers is not in the document]
Comment (sevoku) [reply main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ExceptionCaughtDialog.cs]: yep, here: https://github.com/mono/monodevelop/blob/master/main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/Styles.cs#L46

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/Styles.cs [hunk 2]
```diff
@@ -561,19 +561,19 @@ internal static void LoadLightStyle ()
...
 			NewProjectDialog.TemplateSectionSeparatorColor = Color.FromName ("#e2e2e2");
 			NewProjectDialog.TemplateLanguageButtonBackground = BaseBackgroundColor;
 			NewProjectDialog.ProjectConfigurationSeparatorColor = Color.FromName ("#d2d5d9");
 
 			// Wizards
 
-			Wizard.BannerBackgroundColor = Color.FromName ("#77828c");
-			Wizard.BannerShadowColor = Color.FromName ("#707a83");
-			Wizard.BannerForegroundColor = BaseBackgroundColor;
+			Wizard.BannerBackgroundColor = NewProjectDialog.BannerBackgroundColor;
```
Comment (sevoku) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/Styles.cs]: The `Wizard` style should not depend on the `NewProjectDialog` style, it should be vice versa.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/Styles.cs [hunk 3]
```diff
@@ -682,19 +682,19 @@ internal static void LoadDarkStyle ()
...
 			NewProjectDialog.TemplateSectionSeparatorColor = ThinSplitterColor;
 			NewProjectDialog.TemplateLanguageButtonBackground = SecondaryBackgroundDarkerColor;
 			NewProjectDialog.ProjectConfigurationSeparatorColor = Color.FromName ("#6e6e6e");
 
 			// Wizards
 
-			Wizard.BannerBackgroundColor = Color.FromName ("#6a737e");
-			Wizard.BannerShadowColor = Wizard.BannerBackgroundColor;
-			Wizard.BannerForegroundColor = Color.FromName ("#ffffff");
+			Wizard.BannerBackgroundColor = NewProjectDialog.BannerBackgroundColor;
```
Comment (sevoku) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/Styles.cs]: The `Wizard` style should not depend on the `NewProjectDialog` style, it should be vice versa.

**Comment by slluis** [comment]
@monojenkins backport release-7.7

**Pull request merged**